Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -68,6 +68,23 @@ class SiteTableViewHeader : UITableViewHeaderFooterView {
     }
 }
 
+class HistoryTableViewHeader: SiteTableViewHeader {
```

Comment:
Nit: I would start breaking this out into another file instead of overloading `SiteTableViewController.swift`